---
title: "Dynamic data-driven meta-analysis for prioritisation of host genes implicated in COVID-19"
author: Nicholas Parkinson, Natasha Rodgers, Max Head Fourman, Bo Wang, Marie Zechner, Maaike C. Swets, Jonathan E. Millar, Andy Law, Clark D. Russell, J. Kenneth Baillie & Sara Clohisey
date: '2020-12-18'
slug: scirep-dynamic-data-drive-meta-analysis
categories:
  - Research
tags:

doi: 'https://doi.org/10.1038/s41598-020-79033-3'
publishDate: '2020-12-18'
publication_types:
  - '2'
publication: 'Scientific Reports'
publication_short: 'Sci. Rep.'
abstract: 'The increasing body of literature describing the role of host factors in COVID-19 pathogenesis demonstrates the need to combine diverse, multi-omic data to evaluate and substantiate the most robust evidence and inform development of therapies. Here we present a dynamic ranking of host genes implicated in human betacoronavirus infection (SARS-CoV-2, SARS-CoV, MERS-CoV, seasonal coronaviruses). We conducted an extensive systematic review of experiments identifying potential host factors. Gene lists from diverse sources were integrated using Meta-Analysis by Information Content (MAIC). This previously described algorithm uses data-driven gene list weightings to produce a comprehensive ranked list of implicated host genes. From 32 datasets, the top ranked gene was PPIA, encoding cyclophilin A, a druggable target using cyclosporine. Other highly-ranked genes included proposed prognostic factors (CXCL10, CD4, CD3E) and investigational therapeutic targets (IL1A) for COVID-19. Gene rankings also inform the interpretation of COVID-19 GWAS results, implicating FYCO1 over other nearby genes in a disease-associated locus on chromosome 3. Researchers can search and review the gene rankings and the contribution of different experimental methods to gene rank at https://baillielab.net/maic/covid19. As new data are published we will regularly update the list of genes as a resource to inform and prioritise future studies.'

summary: "The SARS-CoV-2 virus hijacks cellular machinery in the human host to replicate. Understanding which host factors it depends on to survive may inform the development of new therapies and help prioritise the investigation of repurposed drugs.
\n\n
Host factors can be identified using a variety of experimental methods (CRISPR screens, Protein interaction screens, Genetic Association Studies). We aimed to identify and combine the results of relevant experiments to produce a prioritised list of implicated host genes.
\n\n
We conducted a systematic literature search of experiments identifying host genes implicated in human betacoronavirus infection. We screened 12,586 articles → identified 213 relevant full text articles → included 31 studies in the original meta analysis.
\n\n
We extracted relevant gene lists and combined the results using Meta Analysis by Information Content (MAIC). MAIC uses data-driven gene list weightings to
produce a comprehensive ranked list of implicated host genes.
"

featured: yes
links:
- icon: doi
  icon_pack: ai
  name: Publication
  url: https://doi.org/10.1038/s41598-020-79033-3
#- icon: newspaper
#  icon_pack: far
#  name: 
#  url: 
#- icon: newspaper
#  icon_pack: far
#  name: 
#  url: 
image:
  caption: ''
  placement: 2
  focal_point: 'Center'
  preview_only: no
projects: []
slides: ''
---

## Abstract 

The increasing body of literature describing the role of host factors in COVID-19 pathogenesis demonstrates the need to combine diverse, multi-omic data to evaluate and substantiate the most robust evidence and inform development of therapies. Here we present a dynamic ranking of host genes implicated in human betacoronavirus infection (SARS-CoV-2, SARS-CoV, MERS-CoV, seasonal coronaviruses). 

We conducted an extensive systematic review of experiments identifying potential host factors. Gene lists from diverse sources were integrated using Meta-Analysis by Information Content (MAIC). This previously described algorithm uses data-driven gene list weightings to produce a comprehensive ranked list of implicated host genes. 

From 32 datasets, the top ranked gene was PPIA, encoding cyclophilin A, a druggable target using cyclosporine. Other highly-ranked genes included proposed prognostic factors (CXCL10, CD4, CD3E) and investigational therapeutic targets (IL1A) for COVID-19. Gene rankings also inform the interpretation of COVID-19 GWAS results, implicating FYCO1 over other nearby genes in a disease-associated locus on chromosome 3. 

Researchers can search and review the gene rankings and the contribution of different experimental methods to gene rank at https://baillielab.net/maic/covid19. As new data are published we will regularly update the list of genes as a resource to inform and prioritise future studies.


![Fig 1](featured.jpg)
