---
title: "Genetic mechanisms of critical illness in COVID-19"
author: "Erola Pairo-Castineira, Sara Clohisey, Lucija Klaric, Andrew D. Bretherick, Konrad Rawlik, Dorota Pasko, Susan Walker, Nick Parkinson, Max Head Fourman, Clark D. Russell, James Furniss, Anne Richmond, Elvina Gountouna, Nicola Wrobel, David Harrison, Bo Wang, Yang Wu, Alison Meynert, Fiona Griffiths, Wilna Oosthuyzen, Athanasios Kousathanas, Loukas Moutsianas, Zhijian Yang, Ranran Zhai, Chenqing Zheng, Graeme Grimes, Rupert Beale, Jonathan Millar, Barbara Shih, Sean Keating, Marie Zechner, Chris Haley, David J. Porteous, Caroline Hayward, Jian Yang, Julian Knight, Charlotte Summers, Manu Shankar-Hari, Paul Klenerman, Lance Turtle, Antonia Ho, Shona C. Moore, Charles Hinds, Peter Horby, Alistair Nichol, David Maslove, Lowell Ling, Danny McAuley, Hugh Montgomery, Timothy Walsh, Alexandre C. Pereira, Alessandra Renieri, The GenOMICC Investigators, The ISARIC4C Investigators, The COVID-19 Human Genetics Initiative, 23andMe Investigators, BRACOVID Investigators, Gen-COVID Investigators, Xia Shen, Chris P. Ponting, Angie Fawkes, Albert Tenesa, Mark Caulfield, Richard Scott, Kathy Rowan, Lee Murphy, Peter J. M. Openshaw, Malcolm G. Semple, Andrew Law, Veronique Vitart, James F. Wilson & J. Kenneth Baillie"
date: '2020-12-11'
slug: nature-genetic-mechanisms-critical-illness
categories:
  - Research
tags:

doi: 'https://doi.org/10.1038/s41586-020-03065-y'
publishDate: '2020-12-11'
publication_types:
  - '2'
publication: 'Nature'
publication_short: 'Nature'
abstract: 'Host-mediated lung inflammation is present, and drives mortality, in the critical illness caused by coronavirus disease 2019 (COVID-19). Host genetic variants associated with critical illness may identify mechanistic targets for therapeutic development3. Here we report the results of the GenOMICC (Genetics Of Mortality In Critical Care) genome-wide association study in 2,244 critically ill patients with COVID-19 from 208 UK intensive care units. We have identified and replicated the following new genome-wide significant associations: on chromosome 12q24.13 (rs10735079, P = 1.65 × 10−8) in a gene cluster that encodes antiviral restriction enzyme activators (OAS1, OAS2 and OAS3); on chromosome 19p13.2 (rs74956615, P = 2.3 × 10−8) near the gene that encodes tyrosine kinase 2 (TYK2); on chromosome 19p13.3 (rs2109069, P = 3.98 ×  10−12) within the gene that encodes dipeptidyl peptidase 9 (DPP9); and on chromosome 21q22.1 (rs2236757, P = 4.99 × 10−8) in the interferon receptor gene IFNAR2. We identified potential targets for repurposing of licensed medications: using Mendelian randomization, we found evidence that low expression of IFNAR2, or high expression of TYK2, are associated with life-threatening disease; and transcriptome-wide association in lung tissue revealed that high expression of the monocyte–macrophage chemotactic receptor CCR2 is associated with severe COVID-19. Our results identify robust genetic signals relating to key host antiviral defence mechanisms and mediators of inflammatory organ damage in COVID-19. Both mechanisms may be amenable to targeted treatment with existing drugs. However, large-scale randomized clinical trials will be essential before any change to clinical practice.'
summary: 'My contribution to this publication was conducting gene-level and pathway analyses using the bioinformatics tool: MAGMA. SNPs were mapped to genes if the SNPs location was within 5 kb up- or down-stream of the gene region (transcription start site -  transcription stop site). The sum of squared SNP Z-statistics was used as the test statistic. The European reference panel of the 1000 Genomes Project was used to estimate LD between SNPs.'

featured: yes
links:
- icon: doi
  icon_pack: ai
  name: Publication
  url: https://doi.org/10.1038/s41586-020-03065-y
#- icon: newspaper
#  icon_pack: far
#  name: 
#  url: 
#- icon: newspaper
#  icon_pack: far
#  name: 
#  url: 
image:
  caption: ''
  placement: 2
  focal_point: 'Center'
  preview_only: no
projects: []
slides: ''
---

## Abstract 

Host-mediated lung inflammation is present, and drives mortality, in the critical illness caused by coronavirus disease 2019 (COVID-19). Host genetic variants associated with critical illness may identify mechanistic targets for therapeutic development. Here we report the results of the GenOMICC (Genetics Of Mortality In Critical Care) genome-wide association study in 2,244 critically ill patients with COVID-19 from 208 UK intensive care units. We have identified and replicated the following new genome-wide significant associations: on chromosome 12q24.13 (rs10735079, P = 1.65 × 10−8) in a gene cluster that encodes antiviral restriction enzyme activators (OAS1, OAS2 and OAS3); on chromosome 19p13.2 (rs74956615, P = 2.3 × 10−8) near the gene that encodes tyrosine kinase 2 (TYK2); on chromosome 19p13.3 (rs2109069, P = 3.98 ×  10−12) within the gene that encodes dipeptidyl peptidase 9 (DPP9); and on chromosome 21q22.1 (rs2236757, P = 4.99 × 10−8) in the interferon receptor gene IFNAR2. We identified potential targets for repurposing of licensed medications: using Mendelian randomization, we found evidence that low expression of IFNAR2, or high expression of TYK2, are associated with life-threatening disease; and transcriptome-wide association in lung tissue revealed that high expression of the monocyte–macrophage chemotactic receptor CCR2 is associated with severe COVID-19. Our results identify robust genetic signals relating to key host antiviral defence mechanisms and mediators of inflammatory organ damage in COVID-19. Both mechanisms may be amenable to targeted treatment with existing drugs. However, large-scale randomized clinical trials will be essential before any change to clinical practice.

![Fig 1](featured.jpg)
